common/src/main/scala/explore/common/UserPreferencesQueries.scala

```diff
@@ -60,6 +66,23 @@ object UserPreferencesQueries {
           }
         }
    """
+
+    def storeWidthPreference[F[_]: Effect](
+      userId:  Option[User.Id],
+      section: ResizableSection,
+      width:   Int
+    )(implicit
+      cl:      GraphQLClient[F, UserPreferencesDB]
+    ): Callback =
```

Comment:
Just a matter of having a consistent style: in other `*Queries` objects, there's nothing react specific, with the methods returning `IO`s, and the conversion to `Callback`s being done on the components that call the queries.

Reply:
Ok, that sounds a good practice, these should return `IO`